topo_processor/util/conversions.py

```diff
@@ -24,7 +24,7 @@ def remove_empty_strings(properties: Dict) -> Dict:
     return {key: value for key, value in properties.items() if value != ""}
 
 
-def string_to_boolean(value, true_values, false_values):
+def string_to_boolean(value: str, true_values: list, false_values: list) -> Union[bool, str]:
```

Comment:
They might be a `list` of `str`. You want to use `List[str]` in this case. (`List` is the annotation type for `list`)